website/docs/r/netapp_pool.html.markdown

```diff
@@ -0,0 +1,69 @@
+---
+subcategory: "NetApp"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_netapp_pool"
+sidebar_current: "docs-azurerm-resource-netapp-pool"
+description: |-
+  Manages a NetApp Pool.
+---
+
+# azurerm_netapp_pool
+
+Manages a NetApp Pool.
+
+
+## NetApp Pool Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_netapp_account" "example" {
+  name                = "example-netappaccount"
+  location            = "${azurerm_resource_group.example.location}"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+}
+
+resource "azurerm_netapp_pool" "example" {
+  name                = "example-netapppool"
+  account_name        = "${azurerm_netapp_account.example.name}"
+  location            = "${azurerm_resource_group.example.location}"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+  service_level       = "Premium"
+  size                = "4398046511104"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the NetApp Pool. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group where the NetApp Pool should be created. Changing this forces a new resource to be created.
+
+* `account_name` - (Required) The name of the NetApp account.
+
+* `location` - (Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.
+
+* `service_level` - (Required) The service level of the file system.
```

Comment:
We should note what the allowed values are:

```suggestion
* `service_level` - (Required) The service level of the file system. Valid values include `Premium`, `Standard`, or `Ultra`.
```

Reply:
Fixed